src/environments/environments/CarTrackEnvironment.py

```diff
@@ -65,22 +68,54 @@ def __init__(self,
         super().__init__('car_track', car_name, max_steps, step_length)
 
         
-
+        #
         #####################################################################################################################
         # CHANGE SETTINGS HERE, might be specific to environment, therefore not moved to config file (for now at least).
 
         # Reward configuration
         self.BASE_REWARD_FUNCTION:Literal["goal_hitting", "progressive"] = 'progressive'
         self.EXTRA_REWARD_TERMS:List[Literal['penalize_turn']] = []
-        self.REWARD_MODIFIERS:List[Tuple[Literal['turn','wall_proximity'],float]] = [('turn', 0.3), ('wall_proximity', 0.7)] # [ (penalize_turn", 0.3), (penalize_wall_proximity, 0.7) ]
+        self.REWARD_MODIFIERS:List[Tuple[Literal['turn','wall_proximity','lin_acc'],float]] = [('turn', 0.2), ('wall_proximity', 0.6), ('lin_acc',0.2)] # [ (penalize_turn", 0.3), (penalize_wall_proximity, 0.7) ]
```

Comment:
Not sure if it's just github showing it weird but if not this needs to be indented